index.d.ts

```diff
@@ -4,8 +4,12 @@ import { Component } from 'react';
 
 export interface WebViewProps extends React.HTMLAttributes<Electron.WebViewElement>, React.ClassAttributes<WebView> {
   src: string
-  
-  autosize?: boolean
+
+  autosize?: string;
```

Comment:
Does this need to be a string? Seems like it is in the Electron docs but I'm pretty sure just the presence of this attribute will trigger it.